lumicks/pylake/file.py

```diff
@@ -53,24 +52,26 @@ def _check_file_format(self):
         if ff_version not in File.SUPPORTED_FILE_FORMAT_VERSIONS:
             raise Exception(f"Unsupported Bluelake file format version {ff_version}")
 
-        # List of fields which should not be accessed directly from the h5 and their actual API accessors. Note that
-        # top level fields are automatically printed when print is invoked.
-        self.redirect_list = OrderedDict(
-            [
-                ("Calibration", "force1x.calibration"),
-                ("Marker", "markers"),
-                ("FD Curve", "fdcurves"),
-                ("Kymograph", "kymos"),
-                ("Scan", "scans"),
-                ("Note", "notes"),
-            ]
-        )
+        # List of h5 fields for which custom API exists. Note that top level fields (provided as
+        # the first argument in the tuple) are automatically printed when print is invoked. For
+        # fields where an actual class exists, the second element in the tuple is used to
+        # instantiate an element in that Group (using its `from_dataset` method).
```

Comment:
just a suggestion if you think it's warranted 
```suggestion
        # List of h5 fields for which custom API exists. Note that top level fields (provided as
        # the first argument in the tuple) are automatically printed when print is invoked. For
        # fields where an actual class exists, the second element in the tuple is used to
        # instantiate an element in that Group (using its `from_dataset` method). Otherwise
        # a FutureWarning is raised.
```

Reply:
It's good to be explicit about it indeed. I've added it.